models/jules/R/write.config.JULES.R

```diff
@@ -585,6 +590,33 @@ write.config.JULES <- function(defaults, trait.values, settings, run.id) {
     ic.text <- readLines(con = ic.dat, n = -1)
     ic.text[2] <- paste(ic.text[2]," 5.0 5.0 0.5 0.5 0.5 0.2 0.2 0.2 0.2 0.2 0.0 0.0 0.0 0.0")
     writeLines(ic.text, con = ic.dat)
+    
+    } else if (!is.null(settings$run$inputs$poolinitcond$path)){
+      
+      #Read in path to standard initial conditions file
+      IC.path <- settings$run$inputs$poolinitcond$path
+      
+      ## LAI m2/m2
+      if(!is.null(IC.path)){
+        # Read in netcdf file
+        IC.nc <- ncdf4::nc_open(IC.path) 
+        #Get LAI value
+        lai <- try(ncdf4::ncvar_get(IC.nc,"LAI"))
+        #Read in default initial conditions dat file
+        ic.dat  <- file.path(local.rundir, "initial_conditions.dat")
+        ic.dat.text <- readLines(con = ic.dat, n = -1)
+        #For now if number of LAI values does not match number of pfts set them set them across
+        if (npfts!= length(lai)) {
+          ic.dat.text <- gsub("@LAI_DAT@", paste0(rep(lai,npfts), collapse = " "), ic.dat.text)
```

Comment:
The approach taken of subbing in a match against this tag will fail because most cases currently leave the tag in place. Specifically, the previous code above, and the non-TRIFFID case, as well as the case here were LAI isn't in the IC file. In addition, if the user passes in a non-default reference case this tag won't be present. Given that the text file isn't actually hard to parse, I'd ditch the tag approach.